### PR TITLE
Feature/white voting

### PR DESCRIPTION
**Pull request by samrodman**
This feature allows the decide system to create a default option equal to a blank vote whenever we create a new question, even when the question's options are empty.

The way to achieve this, when you create a voting as a staff member, in the moment of creating a question, you can tick the voteBlank check and the blank ballot will be automatically added as the first option in the options of the question.  It will only be added if it doesn't already detect another blank ballot in the options.

If you don't tick the voteBlank option, any other blank votes manually written in will not be tallied. You must tick the option for voting in blank if you wish for it to appear and count.

### CODE REVIEW

**Review by marnunrey2 — APPROVED** [review]
Everything looks fine and working

**Comment by samrodman** [comment]
This feature and its branch was originally created and started by user FelixoGudiel, but after some time he moved to another feature and I continued this one. I also implemented the tests like originally planned.

**Comment by TunixR** [comment]
> This feature and its branch was originally created and started by user FelixoGudiel, but after some time he moved to another feature and I continued this one. I also implemented the tests like originally planned.

I understand, it was not ideal since we decided that the tests would be done by a different person from the one implementing the functionality but I get the situation and it's no problem. All good :+1:

File: decide/voting/models.py [hunk 1]
```diff
@@ -124,6 +145,7 @@ def tally_votes(self, token=""):
             pass
 
         self.tally = response.json()
+        print(self.tally)
```
Comment (TunixR) [inline decide/voting/models.py]: Please remove this print statement

**Review by AntonioRodriguezRuiz — CHANGES_REQUESTED** [review]

**Review by AntonioRodriguezRuiz — CHANGES_REQUESTED** [review]
I found that if you try to create a white voting from the django admin panel and assign value 1 to the first voting, which should be the defaiult. The resulting question ends up like this:
image

Which could be a problem on the tally. I suggest that you make the question number the last

**Comment by samrodman** [comment]
changes have been implemented. should work fine now, thanks.

**Pull request merged**